pkg/http/api.go

```diff
@@ -138,12 +138,15 @@ func RevokeAuthToken(host string, verifyTLS bool, token string) (map[string]inte
 }
 
 // DownloadSecrets for specified project and config
-func DownloadSecrets(host string, verifyTLS bool, apiKey string, project string, config string, format models.SecretsFormat, nameTransformer *models.SecretsNameTransformer, etag string, dynamicSecretsTTL time.Duration) (int, http.Header, []byte, Error) {
+func DownloadSecrets(host string, verifyTLS bool, apiKey string, project string, config string, format models.SecretsFormat, nameTransformer *models.SecretsNameTransformer, etag string, dynamicSecretsTTL time.Duration, secrets []string) (int, http.Header, []byte, Error) {
 	var params []queryParam
 	params = append(params, queryParam{Key: "project", Value: project})
 	params = append(params, queryParam{Key: "config", Value: config})
 	params = append(params, queryParam{Key: "format", Value: format.String()})
 	params = append(params, queryParam{Key: "include_dynamic_secrets", Value: "true"})
+	if len(secrets) > 0 {
+		params = append(params, queryParam{Key: "secrets", Value: strings.Join(secrets, ",")})
+	}
```

Comment:
This might be a big refactor, but we could leverage the `url` stdlib go package for query params, URL encoding and more. It's the same idea as the `new URL(...)` from js (I made a similar comment on the notes PR: https://github.com/DopplerHQ/cli/pull/352#discussion_r1052618951

Reply:
+1 that's a great call out. I do think that's a bigger refactor but will create a linear ticket for it. I'd love to also move to passing around a `context` object, which will help further consolidate a bunch of args we're shuffling around.